Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook

```diff
@@ -403,7 +403,7 @@
                           }
                         },
                         {
-                          "columnMatch": "-- Group By --",
+                          "columnMatch": "$gen_group",
```

Comment:
I don't know what this change would affect, probably better to keep the "-- Group By --"

Reply:
added: -- Group By --